aws_lambda_powertools/logging/logger.py

```diff
@@ -89,18 +90,15 @@ class Logger(logging.Logger):
                 logger.structure_logs(append=True, payment_id=event["payment_id"])
                 logger.info("Hello")
 
-    Parameters
-    ----------
-    logging : logging.Logger
-        Inherits Logger
```

Comment:
No, because I'm proxying all not found attributes directly to the inner logger. It's the same effect of sub-classing for advanced customers who were making changes directly to the logger afterwards , except done right now